src/include/cephfs/metrics/Types.h

```diff
@@ -53,7 +53,8 @@ inline std::ostream &operator<<(std::ostream &os, const ClientMetricType &type)
     os << "OPENED_INODES";
     break;
   default:
-    os << "Unknown metric type: " << type;
+    os << "(UNKNOWN)";
+    break;
```

Comment:
Is it possible to still print whatever integer type this metric was? Could we do something like this?

```os << "Unknown metric type: " << (unsigned int)type;```

...that might be better for troubleshooting later.

Reply:
done!